package.json

```diff
@@ -10,10 +9,41 @@
     "build-tx-builder": "dotenv -e .env -- yarn workspace tx-builder build",
     "build-wallet-connect": "dotenv -e .env -- yarn workspace wallet-connect build"
   },
+  "version": "2.0.0",
   "workspaces": [
     "apps/*"
   ],
+  "dependencies": {
```

Comment:
I don't have an strong opinion on devDependencies, as they aren't packed when compiling the app.

I see that this dependencies are used in the three apps, but if we do this we should be careful when adding new apps. If an app is not using react, react will be packed anyway.

This also can add difficulties when upgrading dependencies and breaking changes. If a dependency upgrade is critical for one app but we have breaking changes with other app we should shift again to have dependencies at the app level if we want to deliver a hotfix without upgrading both apps.

Just wanted to comment this because that were the reasons of leaving it like that before.

Reply:
>  If an app is not using react, react will be packed anyway.

I don't think so, the bundler will only include what's imported in the app.

> This also can add difficulties when upgrading dependencies and breaking changes. If a dependency upgrade is critical for one app but we have breaking changes with other app we should shift again to have dependencies at the app level if we want to deliver a hotfix without upgrading both apps.

True. But apps are pretty small, and, for example, updating the SDK for every app was pretty much changing only a few lines of code. Adding a dependency on the app level (for the case you mentioned) is not that hard (just running 1 command basically), so I think it's fine. React.js rarely introduces breaking changes. I did the change mostly to motivate us to keep the dependencies up-to-date 😄

Reply:
This was a requirement by me. Updating dependencies for all apps at the same time broke functionality before.

If I remember correctly we now publish everything via ipfs so it is less critical, but in general it should be possible to update/ add dependencies for one of the Safe apps without touching the other Safe apps.

Reply:
It's still possible. Only sdk, react and styled components are shared. But locking a dependency for one app is as easy as running `yarn workspaces *app-name* add dep@v1`